### PR TITLE
Feat/Added naive timeline. 

### LINKED ISSUE #70: User Profile Map

### PR DESCRIPTION
**Pull request by Sam0064**
Closes #70

### CODE REVIEW

**Comment by Sam0064** [comment]
Forgot to pull down latest changes.

File: client/src/components/main/UserProfilePage.jsx [hunk 1]
```diff
@@ -30,6 +32,10 @@ const UserProfilePage = (props) => {
           </div>
         </Col>
       </Row>
+      <Row style={{height: '500px'}}>
+        <Timeline />
+      </Row>
+        <PostList type={'TYPE_STORY'} />
```
Comment (kevinzhuang10) [inline client/src/components/main/UserProfilePage.jsx]: The indent here is funky

File: client/src/components/main/RootRouter.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import Home from './Home.jsx';
 import PostPage from './PostPage.jsx';
 import CreateNewPost from './CreateNewPost.jsx';
 import UserProfilePage from './UserProfilePage.jsx';
+import Timeline from './Timeline.jsx';
```
Comment (kevinzhuang10) [inline client/src/components/main/RootRouter.jsx]: why is this imported?

File: client/src/components/main/Timeline.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+} from '../../store/modules/map';
+
+//The strategy utilized here only works with posts coming from the server organized by timestamp,
+//it cannot use the same routes as the home page.
+
+
+const TimelineComponent = withGoogleMap(props => {
+  const path = [];
+  props.markers.forEach(function (post) {
+    var obj = {'lat': Number(post.lat), 'lng': Number(post.lng)};
```
Comment (dbaker13) [inline client/src/components/main/Timeline.jsx]: Maybe consider standardizing our way of treating these lat/long strings, either Number() or parseFloat() but not both...

**Review by lazdinst — DISMISSED** [review]

File: client/src/components/main/Timeline.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+
+const mapStateToProps = state => ({
+  center: state.map.center,
+  bounds: state.map.bounds,
+  containerElement: <div style={{height: '100%'}} />,
+  mapElement: <div style={{height: '100%'}} />,
+  markers: state.map.markers,
+  map: state.map._map
```
Comment (kevinzhuang10) [inline client/src/components/main/Timeline.jsx]: Please remove. Not there anymore

**Review by kevinzhuang10 — DISMISSED** [review]

**Comment by Sam0064** [comment]
Closing due to Travis Errors.

**Pull request closed without merging**